libs/wlgen/wlgen/rta.py

```diff
@@ -709,30 +737,25 @@ def __init__(self, start_pct=100, end_pct=0, time_s=1, period_ms=100,
 
 
 class Periodic(Pulse):
+    """
+    Configure a periodic load. This is the simplest type of RTA task.
 
-    def __init__(self, duty_cycle_pct=50, duration_s=1, period_ms=100,
-                 delay_s=0, sched=None, cpus=None):
-        """
-        Configure a periodic load.
+    This class defines a task which load is periodic with a configured
+    period and duty-cycle.
 
-        This class defines a task which load is periodic with a configured
-        period and duty-cycle.
+    :param duty_cycle_pct: the load percentage.
+    :param duration_s: the total duration in seconds of the task.
+    :param period_ms: the period used to define the load in milliseconds.
+    :param delay_s: the delay in seconds before starting the workload.
```

Comment:
Technically the workload is started immediately... this is better:
"the delay in seconds before starting the periodic phase"